### ISSUE TITLE
Any reason not to add this to extensions.gnome.org?

### ISSUE THREAD

**Issue by pedrocr** [kazysmaster/gnome-shell-extension-disable-unredirect#11]
There's this similar extension:

https://extensions.gnome.org/extension/1445/fix-fullscreen-tearing/

that's currently broken in 3.32 and includes a button which doesn't seem useful. From the code this does just what is needed. Why not submit it to the site?

**Comment by kazysmaster** [comment]
:) is it worthy enough for adding it to the extensions.gnome.org ?

**Comment by TingPing** [comment]
People use it, so sure why not.

**Comment by ghost** [comment]
So if I installed it directly from https://extensions.gnome.org/extension/1873/disable-unredirect-fullscreen-windows/ it should be ok right? I didn't follow the readme guide, just clicked install in Gnome extensions website. Problem seems fixed, but I just want to be sure

**Comment by tjohnman** [comment]
> :) is it worthy enough for adding it to the extensions.gnome.org ?

This fixes fullscreen tearing in mpv as well. I'd say it's an essential extension for Gnome.

**Comment by PranavBhattarai** [comment]
Yeah please do add this extension in extensions.gnome.org. 

It will be very helpful for noob users like me,

**Comment by Espionage724** [comment]
This extension was the fix I needed to finally have Old School RuneScape working ideally on Intel graphics!

The Jagex Launcher (Webview2/CEF/whatever) only shows "sometimes" on GNOME on Wayland. No idea why it's only sometimes, and it even seems to get to a point where it doesn't show at all without redoing the entire prefix or disabling WM control of it (either the winecfg option specifically or virtual desktop). I only used the Wayland session at all because when OSRS was able to be ran, it ran without any tearing and no need of this extension.

Jagex Launcher starts consistently without issue on GNOME on Xorg, but I had noticeable screen tearing with the OSRS clients (Java and C++) while fullscreened with either the in-game option (C++) or `wmctrl` (Java) around 1/3rd or 1/2 the screen down from the top.

This extension allows OSRS C++ to be fullscreen without tearing on Xorg! I was ready to just give up and stick to playing this on Android devices until I remembered this extension existed. I'm on Fedora 38, GNOME 44.3, and Intel UHD 630.
